tests/console/suseconnect_proxyscc.pm

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 SUSE LLC
```

Comment:
we need to move all our checks to a folder under yam/ wasn't one called validations/ I don't remember.
the goal would be reorganize them later and doing some refactor for them.

Reply:
I moved it to `tests/yam/validate/` and named the testmodule as `post_registration.pm`, if need to rename it, feel free to let me know, thanks.